WaysofWork.md

```diff
@@ -297,5 +289,52 @@ it is the time to elevate blockers or important changes to the roadmap or backlo
 As with weeklys and to make use of the synchronous times this meetings have a strong asynchronous component,
 each team will update their stories and make changes to a monthly report PR that the RPM will prepare in advance.
 
+
+#### Planning & Report (meeting outputs)
+
+These issues will be primarly maintained by the RPM and be used to reflect the planned work and progress made either weekly, monthly or quarterly.
+Monthly and quarterly issues will have associated PRs in which the team is expected to take part.
+
+##### Weekly issue
+
+Purpose of this issue is to better communicate and coordinate with DSH team members directly working on project tasks.
+
+It should reflect what was done during the week, blockers and serve to plan work for the following week minimising the need for synchronous meeting time
+
+RPM will be responsible to create and update it each week,
+everyone is responsible to update issues worked on and planned and report in the collaborative note here
+
+The issue will be permanent and updated via comments to avoid drowing the repo, meeting minutes will not be reflected here but stored in Sharepoint and as files in this repo. This issue will therefore only track status of issues so PM can maintian the team's Project.
+
+From Thursdays and before the weekly meeting (Monday 10 AM) each team member will have to:
+- Make updates in the issues they have worked on via comments
+- Change their status in the project where appropiate
+- Update fields where appropiate
+- Fill in the meeting collaborative note here: https://hackmd.io/nPewceMuQcmXD7jfqYFpzw?edit
+
+The weekly standup will then take place, discussing blockers and planning work.
+Team members should update the collaborative note as appropiate to reflect plans for the week and any agreed actions
+
+The RPM will:
+- Update the weekly issue to reflect the team overall work for the last week and plans for this week
+- Review roadmap and boards to ensure it reflects the current project status
+- Add any necessary points for discussion for the monthly meeting
+
+A member of the REG team will be asked to archive the meeting minute in the collaborative note in Sharepoint [here](https://thealanturininstitute.sharepoint.com/sites/SafeHaven/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FSafeHaven%2FShared%20Documents%2FMeetings&viewid=10151919%2Deeef%2D4a8b%2Db4c0%2D26a3b155773b)
+
+##### Monthly issue
+
+The monthly issue will be used to update the [month by month report](https://github.com/alan-turing-institute/data-safe-haven-team/blob/main/Reports/Monthly.md),
+as well as put together the agenda for the monthly meeting.
+
+After the monthly meeting the RPM will reflect in the issue any necessary updates to roadmap, stories or issues.
+It will allocate responsibility across teams to make those updates effective.
+
+```
+Potentially monthly issues will be reduced to their PR without creating an issue to keep the repo cleaner.
+
+```
```

Comment:
```suggestion
Potentially monthly issues will be reduced to their PR without creating an issue to keep the repo cleaner.
```

Reply:
this is not actually happening, may take out. We ended up not creating issues for meetings

Reply:
this whole section needs revisiting, what is described here happens in the weekly collaborative note. RPM then uses minutes to update stories, then I should (but have no done) create Reports and PRs when appropiatte